docs/pages/contracts/affiliate-program/overview.md

```diff
@@ -0,0 +1,18 @@
+---
+description: Fork of Curve Finance
```

Comment:
in this description intended for Affiliate program?

Reply:
<img width="780" alt="Screenshot 2024-04-24 at 2 38 32 PM" src="https://github.com/pancakeswap/pancake-developer/assets/133624774/a0eb2651-4f45-445d-91b9-dd1b62817bb9">

this is how the page intended, is it expected to have `Stable Swap` keyword too?

Reply:
got it.